Peer.Domain/Configuration/CommandConfigs/ShowConfig.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace Peer.Domain.Configuration.CommandConfigs
+{
+    public class ShowConfig
+    {
+        private const int _defaultRetryCutoffSeconds = 10;
```

Comment:
I'm good to drop MaxReqRetries. TBH I don't like it either; partially for the reason you mention and partially because I don't think it really offers a useful control mechanism.

I think RetryCutoff is nice because it gives you control over how long the command can take and I don't think it imposes many limits since the only implementation detail it leaks is that fact that there _are_ retries on fetching.

I think we should have _something_ for watch; if not MaxFailures then maybe a time based thing. I think MaxFailures works well because its retry for the whole show command so all it tells you about the impl is that we're going to run the command more than once (which is already part of the expected behaviour of watch). ICBC tha a fixed value like 5 is sufficient, but I don't think this really imposes limits either.